src/config/Config.zig

```diff
@@ -563,11 +563,27 @@ palette: Palette = .{},
 @"background-opacity": f64 = 1.0,
 
 /// A positive value enables blurring of the background when background-opacity
-/// is less than 1. The value is the blur radius to apply. A value of 20
+/// is less than 1.
+///
+/// On macOS, the value is the blur radius to apply. A value of 20
 /// is reasonable for a good looking blur. Higher values will cause strange
 /// rendering issues as well as performance issues.
 ///
-/// This is only supported on macOS.
+/// On KDE Plasma under Wayland, the exact value is _ignored_ —  the reason is
+/// that KWin, the window compositor powering Plasma, only has one global blur
+/// setting and does not allow applications to have individual blur settings.
+///
+/// To configure KWin's global blur setting, open System Settings and go to
+/// "Apps & Windows" > "Window Management" > "Desktop Effects" and select the
+/// "Blur" plugin. If disabled, enable it by ticking the checkbox to the left.
+/// Then click on the "Configure" button and there will be two sliders that
+/// allow you to set background blur and noise strengths for all apps,
+/// including Ghostty.
+///
+/// All other Linux desktop environments are as of now unsupported. Users may
+/// need to set environment-specific settings and/or install third-party plugins
+/// in order to support background blur, as there isn't a unified interface for
+/// doing so.
 @"background-blur-radius": u8 = 0,
```

Comment:
I think that it would be better to introduce a `background-blur` boolean to enable/disable background blur and then `background-blur-radius` would then only have an effect on macOS. Or make `background-blur-radius` an optional that defaults to `null`, but I think that the explicit boolean would be clearer.

Reply:
I disagree about introducing a new config. macOS uses 0 to disable blur. We should do the same here. The uglier part is that the magnitude of value doesn't matter for Linux.

I'd propose introduce a new type here `BackgroundBlurRadius` that wraps a `u8` but parses `false` to 0 and `true` to something else (I recommend 20, since that's a reasonable value for macOS). That way this works on both macOS and Linux with "true" but also can be set to a specific number to have an effect on macOS and still enable on Linux.

And ultimately, this keeps a single, backwards compatible config.

Reply:
The config key's name is definitely a bit awkward though... `background-blur-radius = true` doesn't *quite* make sense

Reply:
That's right. Let me think about this a bit more. I'm not ready to accept two configs.

Reply:
To me the best option is to just have one option called `background-blur`: `background-blur = true` and `background-blur = 20` both make sense IMO

Reply:
I agree with this. I think we need to introduce a mechanism to rename. I can work on that. For the sake of this PR, please override `background-blur-radius`. We'll add the mechanism to rename and do the rename in a follow up PR. I don't want to block this for you.